examples.md

```diff
@@ -41,6 +42,26 @@
 - [Swift - Mint](#swift---mint)
 - [* - Bazel](#---bazel)
 
+## Bun
+
+```yaml
+- uses: actions/cache@v4
+  with:
+    path: |
+      ~/.bun/install/cache
```

Comment:
Why not add `node_modules` to the list of paths, @idleberg @joshmgross?

Reply:
@PaulRBerg `node_modules` with Bun is just hardlinks to the aforementioned `~/.bun/install/cache` like PNPM to use less space on disk.

However Bun is usually faster than the cache system, so I wouldn't recommend using it: https://github.com/oven-sh/setup-bun/issues/14#issuecomment-1714116221

Reply:
haha interesting, thanks for your answer!